sign_portal_oca/controllers/portal.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2025 Kencove
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.http import request, route
+
+from odoo.addons.portal.controllers.portal import pager as portal_pager
+from odoo.addons.sign_oca.controllers.main import PortalSign
+
+
+class PortalSign(PortalSign):
+    def get_sign_requests_domain(self, request):
```

Comment:
I think that if in https://github.com/OCA/sign/blob/dfb532e7314027b4ed0911b652b7c8b9caac7dd3/sign_oca/models/res_users.py#L13 you define the domain in a method (`_get_sign_oca_request_user_domain()` for example) you could remove this one and unify it.